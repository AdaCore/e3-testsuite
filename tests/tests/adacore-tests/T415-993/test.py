import os.path


print("H\\E\\L\\O")
print("foo.exe")
print(">>>", os.path.join(os.getcwd(), "foobar"))
