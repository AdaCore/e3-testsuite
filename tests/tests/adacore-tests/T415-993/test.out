H/E/L/O
foo
>>> foobar
